doc/users/explain/figures.rst

```diff
@@ -0,0 +1,175 @@
+
+.. _creating_figures:
+
+Creating Figures
+================
+
+By far the most common way to create a figure is using the `pyplot` interface.
+As noted in `APIS`, the pyplot interface serves two purposes.  One is to spin
+up the Backend and keep track of GUI windows. The other is a global state for
+Axes and Artists that allow a short form API to plotting methods.  In the example
+above, we only use pyplot for the first purpose, to create the Figure object, ``fig``.
+
+In addition to `.pyplot.Figure`, there are convenience methods to create Axes
+at the same time that the figure is created since these two actions are so commonly
+done at the same time. A simple grid can be achieved with `.pyplot.subplots` (which
+simply wraps `.Figure.subplots`):
+
+.. plot::
+    :include-source:
+
+    fig, axs = plt.subplots(2, 2, figsize=(4, 3), layout='constrained')
+
+More complex grids can be achieved with `.pyplot.subplot_mosaic` (which wraps
+`.Figure.subplot_mosaic`):
+
+.. plot::
+    :include-source:
+
+    fig, axs = plt.subplot_mosaic([['A', 'right'], ['B', 'right']],
+                                  figsize=(4, 3), layout='constrained')
+    for ax_name in axs:
+        axs[ax_name].text(0.5, 0.5, ax_name, ha='center', va='center')
+
+We can also have subfigures inside figures (or other subfigures;
+:ref:`/examples/subplots_axes_and_figures/subfigures` for more details).
+
+It is possible to directly instantiate a `.Figure` instance.  This is usually
+only necessary if you want to create your own GUI application and not use
+the pyplot application.  See  see the
+embedding examples in :ref:`/gallery/user_interfaces/` for examples of
+how to do this.
+
+Figure options
+--------------
+
+There are a few options available when creating figures.  The Figure size on
+the screen is set by *figsize* and *dpi*.  *figsize* is the ``(width, height)``
+of the Figure in inches (or, if prefered, units of 72 typographic points).  *dpi*
+are how many pixels per inch the figure will be rendered at.  To make you Figures
+appear on the screen at the physical size you requested, you should set *dpi*
+to the same *dpi* as your graphics system.  Note that many graphics systems now use
+a "dpi ratio" to specify how many screen pixels are used to represent a graphics
+pixel.  Matplotlib applies the dpi ratio to the *dpi* passed to the figure to make
+it have higher resolution, so you should pass the lower number to the figure.
+
+The *facecolor*, *edgecolor*, *linewidth*, and *frameon* options all change the appearance of the
+figure in expected ways, with *frameon* making the figure transparent if set to *False*.
+
+Finally, the user can specify a layout engine for the figure with the *layout*
+parameter.  Currently Matplotlib supplies
+:ref:`"constrained" </tutorials/intermediate/constrainedlayout_guide>`,
+:ref:`"compressed" <compressed_layout>` and
+`"tight" </tutorials/intermediate/tight_layout_guide>` layout engines.  These
```

Comment:
```suggestion
:ref:`"tight" </tutorials/intermediate/tight_layout_guide>` layout engines.  These
```
(although the link is probably not in correct format, not sure which is though...)